plugins/modules/azure_rm_subnet.py

```diff
@@ -100,6 +100,23 @@
         choices:
             - Enabled
             - Disabled
+    delegations:
+        description:
+            - An array of delegations.
+        type: list
+        suboptions:
+            name:
+                description:
+                    - The name of delegation.
+                required: True
+            serviceName:
```

Comment:
It should have a choices, thank you!

Reply:
added choices and updated sanity test files as requested